draft-ietf-lake-edhoc.md

```diff
@@ -965,7 +996,9 @@ The Initiator SHALL compose message_3 as follows:
 
 * Encode message_3 as a CBOR data item as specified in {{asym-msg3-form}}.
 
-*  Make the connection identifiers (C_I, C_R) and the application algorithms in the selected cipher suite available to the application. The application can now compute PRK_out and derive application keys using the EDHOC-Exporter interface, see {{exporter}}.
+*  Make the connection identifiers (C_I, C_R) and the application algorithms in the selected cipher suite available to the application.
+
+* Compute the transcript hash TH_4 = H( TH_3, PLAINTEXT_3 ) where H() is the EDHOC hash algorithm of the selected cipher suite. The transcript hash TH_4 is a CBOR encoded bstr and the input to the hash function is a CBOR Sequence. The application can now compute PRK_out and derive application keys using the EDHOC-Exporter interface, see {{exporter}}.
 
```

Comment:
It seems like the TH_4 definition is now part of the Initiator processing. Not sure that is good.

Reply:
I'll waited with moving this. Note that since TH is defined in terms of plaintext they can be calculated earlier than with the previous definition.